### DISCUSSION TITLE
Add option to hide volume slider

### DISCUSSION THREAD

**Discussion by petebueh** [jcorporation/myMPD#1164] [Ideas]
When touching the volume slider accidentally, the volume can abruptly get very loud. 

It would be great to have the option to hide the volume slider, so the volume only changes when touching the "+" and "-" buttons.

**Comment by nousernouser** [comment]
+1

Hiding the volume slider would also be useful for people who use MPD/myMPD with an external preamplifier device: then the playback volume is managed directly by the pream and... 

It's my case too. My MPD conf has `replay_gain_handler "none"`, the audio server is connected to an external DAC and I adjust the playback volume on the preamp the DAC is connected to.

Regards

**Reply by jcorporation** [reply]
If you disable the volume control in MPD, myMPD shows no volume settings.
